### DISCUSSION TITLE
html编码

### DISCUSSION THREAD

**Discussion by feiyumozhu** [projectdiscovery/nuclei#4665] [Q&A] [answered]
如何对请求体进行html编码

**Comment by olearycrew** [comment]
From `translate.google.com`

html encoding

How to html encode the request body

**Comment by olearycrew** [comment, marked as answer]
@feiyumozhu you can **URL encode** the body with `url_encode` or **HTML escape** the body with `html_escape` (see https://docs.projectdiscovery.io/templates/reference/helper-functions). Would either of those solve the problem you're looking to solve?

Simpler language & translation from `translate.google.com`

Try `html_escape` or `url_encode`. See https://docs.projectdiscovery.io/templates/reference/helper-functions for more

尝试“html_escape”或“url_encode”。 有关更多信息，请参阅 https://docs.projectdiscovery.io/templates/reference/helper-functions
